docs/site/src/contributor/decisions/2023-01-27-continuous-integration.md

```diff
@@ -0,0 +1,47 @@
+# **Continuous Integration**
+
+|              |            |
+|--------------|------------|
+| Started      | 2022-12-06 |
+| Decided      | 2023-01-27 |
+| Last amended | 2023-04-04 |
+
+## **Why**
+
+The sudden decommission of Hydra forces us to revisit our Continuous Integration (CI) setup.
+
+## **Decision**
+
+We predominantly rely on [Buildkite](https://buildkite.com) as CI system. For Windows, we have to use Github Actions.
```

Comment:
```suggestion
We predominantly rely on [Buildkite](https://buildkite.com) as CI system.
```

We have recently switched to Buildkite on Windows as well. I suppose that this decision needs to be amended?